test/Chainweb/Test/Pact/PactSingleChainTest.hs

```diff
@@ -857,8 +857,13 @@ preInsertCheckTimeoutTest _ reqIO = testCase "preInsertCheckTimeoutTest" $ do
         $ set cbRPC (mkExec' coinV5)
         $ defaultCmd
 
-  rs <- pactPreInsertCheck (V.fromList [txCoinV3, txCoinV4, txCoinV5]) q
-  assertBool ("should be InsertErrorTimedOut but got " ++ show rs) $ V.and $ V.map (== Left InsertErrorTimedOut) rs
+  -- timeouts are tricky to trigger in GH actions.
```

Comment:
I would like it if we branched this behaviour on if we are in GitHub actions or not, since doing it this way could hide unreliability.

```haskell
envVars <- getEnvironment
let inGha = flip any envVars $ \(k, v) ->
       (k == "CI" && v /= "")
       || k == "GITHUB_RUN_ID"
       
```
This should be sufficient for determining if we are in gha

Reply:
I don't actually mind if this is unreliable, as long as it happens at all.